pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -1510,6 +1510,70 @@ PJ_DEF(pj_status_t) pjsua_acc_modify( pjsua_acc_id acc_id,
     return status;
 }
 
+typedef struct pjsua_send_request_data
+{
+    pjsua_acc_id acc_id;
+    void *token;
+} pjsua_send_request_data;
+
+static void on_acc_tsx_state(void *request_data, pjsip_event *event)
```

Comment:
Minor: to avoid possible confusion in the future, perhaps don't use `tsx_state` term too here, `on_send_request` for example?